code/modules/reagents/reagent_containers/medspray.dm

```diff
@@ -110,10 +110,13 @@
 	squirt_amount = 1
 	squirt_mode = 1
 
+//MonkeStation Edit Start
+//Changes Spray Tan into Baja Blast
 /obj/item/reagent_containers/medspray/spraytan
```

Comment:
I would say to just make this a child of medspray called gamerspray, its misleading that the spray tan is actually gamer spray

Reply:
The point of keeping the names the same is to prevent map conflicts.
I'm willing to just change the names, but it will cause minor annoyances each time we have to update maps from upstream.